tests/orchestrator/test_continue_as_new.py

```diff
@@ -13,8 +14,8 @@ def generator_function(context):
     context.continue_as_new("Cause I can")
 
 
-def base_expected_state(output=None) -> OrchestratorState:
-    return OrchestratorState(is_done=False, actions=[], output=output)
+def base_expected_state(output=None, replay_schema: ReplaySchema = ReplaySchema.V1) -> OrchestratorState:
+    return OrchestratorState(is_done=False, actions=[], output=output, replay_schema=replay_schema.value)
```

Comment:
Is there a reason we default this value again, instead of just grabbin it of the parameter `replay_schema` on `base_expected_state`?

Reply:
Isn't that what we're doing? I'm not sure I understand this comment :)

Reply:
woops, I think I saw some subsequent cases doing `replay_schema.V1.value`.

Reply:
Yeah, in general, the unit tests do not do a great job at re-using utilities across files, so there's some repeated logic depending on which testing file you're looking at. We should prioritize doing a refactor of the tests themselves, it's proving to become a minor obstacle throughout project Laguna